### ISSUE TITLE
K.set_learning_phase(0)

### ISSUE THREAD

**Issue by soldierofhell** [qjadud1994/CRNN-Keras#15]
Hi,

I don't get it why in training.py there's a line 7, which force the learning_phase to 0 (=test)?
`K.set_learning_phase(0)`

As far as I know for example BatchNormalization() layer is behaves differently in 0 and 1 phase.

**Comment by qjadud1994** [comment]
In fact, I did not really understand K.set_learning_phase (0).

Since there was too much performance difference according to 0 or 1, I fixed it to 0 and did training and inference.

If anyone knows exactly what K.set_learning_phase () is, please let me know.

**Comment by DeeeepNet** [comment]
K.set_learning_phase (), in the older version of keras, because of BN layers, we must specify the training and inference mode, but in the new version of keras this bug is fixed and don't necessary to set this option.
